reya-rust-sdk/src/data_types.rs

```diff
@@ -16,6 +73,32 @@ pub enum CommandType {
     TransferMarginAccount,
 }
 
+#[allow(dead_code)]
+#[repr(u8)]
+#[derive(PartialEq, Eq, Clone, Copy, Debug)]
+pub enum OrderType {
+    StopLoss = 0,
+    TakeProfit,
+}
+
+/// order struct to execute orders in a batch
+pub struct BatchOrder {
+    pub account_id: u128,
+    pub market_id: u128,
+    pub exchange_id: u128,
+    pub order_type: OrderType,
+    /// side(+/- = buy/sell) + volume i256
+    pub order_base: I256,
```

Comment:
hmm, we don't need order_base

Reply:
this not only for SL, but the batch order can execute more types, so far I understand  and will in the next sprints

Reply:
hmm, yes but i wouldn't keep smth for the future if we don't need it for the current deployment to keep the keep the logic simple